static/app/views/settings/project/tempest/PlayStationSettings.tsx

```diff
@@ -0,0 +1,199 @@
+import {Fragment, useEffect, useMemo} from 'react';
+
+import {addErrorMessage, addSuccessMessage} from 'sentry/actionCreators/indicator';
+import {openAddTempestCredentialsModal} from 'sentry/actionCreators/modal';
+import {Alert} from 'sentry/components/core/alert';
+import {Button} from 'sentry/components/core/button';
+import {ButtonBar} from 'sentry/components/core/button/buttonBar';
+import {Tooltip} from 'sentry/components/core/tooltip';
+import FeedbackWidgetButton from 'sentry/components/feedback/widget/feedbackWidgetButton';
+import Form from 'sentry/components/forms/form';
+import JsonForm from 'sentry/components/forms/jsonForm';
+import List from 'sentry/components/list';
+import ListItem from 'sentry/components/list/listItem';
+import Panel from 'sentry/components/panels/panel';
+import {PanelTable} from 'sentry/components/panels/panelTable';
+import {IconAdd} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import type {Organization} from 'sentry/types/organization';
+import type {Project} from 'sentry/types/project';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import {handleXhrErrorResponse} from 'sentry/utils/handleXhrErrorResponse';
+import {useMutation} from 'sentry/utils/queryClient';
+import type RequestError from 'sentry/utils/requestError/requestError';
+import useApi from 'sentry/utils/useApi';
+import {useFetchTempestCredentials} from 'sentry/views/settings/project/tempest/hooks/useFetchTempestCredentials';
+import {MessageType} from 'sentry/views/settings/project/tempest/types';
+import {useHasTempestWriteAccess} from 'sentry/views/settings/project/tempest/utils/access';
+
+import {CredentialRow} from './CredentialRow';
+import EmptyState from './EmptyState';
+
+interface Props {
+  organization: Organization;
+  project: Project;
+}
+
+export default function PlayStationSettings({organization, project}: Props) {
+  const hasWriteAccess = useHasTempestWriteAccess();
+  const {
+    data: tempestCredentials,
+    isLoading,
+    invalidateCredentialsCache,
+  } = useFetchTempestCredentials(organization, project);
+
+  const api = useApi();
+  const {mutate: handleRemoveCredential, isPending: isRemoving} = useMutation<
+    unknown,
+    RequestError,
+    {id: number}
+  >({
+    mutationFn: ({id}) =>
+      api.requestPromise(
```

Comment:
you can use `fetchMutation` and pass that directly to `mutationFn`. This avoids the need for `usApi`. example:

https://github.com/getsentry/sentry/blob/1b4ce1b3effc0dc560b4f4498fd92e7b98344179/static/app/views/settings/account/notifications/notificationSettingsByType.tsx#L333-L334